qiskit/providers/ibmq/api_v2/rest/backend.py

```diff
@@ -36,10 +37,19 @@ def __init__(self, session, backend_name):
         self.backend_name = backend_name
         super().__init__(session, '/devices/{}'.format(backend_name))
 
-    def properties(self):
+    def properties(self, filter=None):
         """Return backend properties."""
+        # pylint: disable=redefined-builtin
```

Comment:
Is there any reason to disable lint here instead of renaming your variable? It's general better to not use the same name as a builtin.

Reply:
I was using the job filtering functionality as reference. The last function that made the api call used `filter` as the parameter name, and it disabled the builtin warning as well. I know it's best not to use builtin names, it was more to be consistent with other areas of the code. Would it be best to change it to another name perhaps?